### PR TITLE
mailroom part 3, and ungraded unit test exercises

### PR DESCRIPTION
**Pull request by nichx313**
This PR is just for mailroom-pt3.py, unless I finish something else in the meantime and come back to comment. Thanks!

### CODE REVIEW

**Comment by charlesrobison** [comment]
@nichx313 - this is just for mailroom-pt3.py?  Not seeing that file in this PR.  If you're ready for Lesson 6 mailroom, there should be a test file as well as your mailroom code.  Similar to what you have for walnut party.  Is that ready for review?

**Comment by nichx313** [comment]
Pt3 only, yes. Let me make and push a small change briefly.

**Comment by nichx313** [comment]
Ok, pt3 should show up now.

File: students/nick_miller/lesson05/mailroom-pt3.py [hunk 1]
```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env python3
 
-"""PY210_SP - mailroom part 3
+"""PY210_SP - mailroom-part 3
 author: Nick Miller"""
 
 import sys
```
Comment (charlesrobison) [inline students/nick_miller/lesson05/mailroom-pt3.py]: import not being used

**Comment by nichx313** [comment]
Fixed it; I think I may have tested fixed this in a ’sketchpad’ and not rolled the change over. Will re-submit on next PR. Thanks!

**Pull request merged**